AIPscan/templates/report_base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>AIPscan - Report</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/jquery-ui.css') }}">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/bootstrap.min.css') }}">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/custom.css') }}">
```

Comment:
@tw4l I was wondering about this as I work on the agents table. I used a custom CSS, say `style_agents_report.css`. This grants some flexibility around the amount of customization I can do in that table. Minor stuff, but it also leaves out the ambiguity of having to check whether a style is being used in more than one report. 

Does that make sense to you? And if so, can we split `base.html` into two? (although I am not sure how that would with the need for a `</head>` end tag for example. 

Or should I perhaps use `custom.css` and just use comments to separate sections, but accept that stylesheet might grow over time? 

It looks like I am coming to a pause on this one tonight, so will get back to it tomorrow.

Reply:
Hi @ross-spencer - this is a good question!

We could use a single `custom.css` as you say, but another option might be to use an additional block in order to add custom CSS per report.

Looking at Archivematica as an example, we could do something like this in the `report_base.html` template: https://github.com/artefactual/archivematica/blob/stable/1.12.x/src/dashboard/src/templates/layout.html#L14

And then in our reports templates, add the additional CSS like this: https://github.com/artefactual/archivematica/blob/stable/1.12.x/src/dashboard/src/templates/archival_storage/archival_storage.html#L19-L24

Those are Django templates and not jinja2, of course, but I think it should work the same way.

Reply:
Doing the same for the jQuery/JS sprinkled into the templates might be a nice way to cleanly separate that out as well.